locale/it/about/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: about.hbs
+title: About
+trademark: Trademark
+---
+# Riguardo a Node.js&reg;
+
+In quanto runtime JavaScript guidato dagli eventi, Node è pensato per creare applicazioni di rete scalabili. Nell'esempio "hello world" seguente, possono essere gestite molte connessioni simultaneamente. Nel momento in cui la callback di ogni connessione è richiamata, ma non c'è lavoro da svolgere, Node rimarrà dormiente.
+
+```javascript
+const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World\n');
+});
+
+server.listen(port, hostname, () => {
+  console.log(`Server in esecuzione all'indirizzo http://${hostname}:${port}/`);
+});
+```
+Ciò è in contrasto con il modello comune del giorno d'oggi, in cui vengono sfruttati i thread del sistema operativo. Il networking basato sui thread è piuttosto inefficiente e molto difficile da usare. Inoltre, gli utenti di Node sono liberi da preoccupazioni di processi che entrano in dead-lock, dal momento in cui non ci sono blocchi. Quasi nessuna funzione in Node esegue direttamente operazioni di I/O, perciò il processo non si blocca mai. Dal momento che nulla si blocca, è molto facile sviluppare sistemi scalabili in Node.
+
+Se qualcosa in questa terminologia ti è poco familiare, qui c'è un articolo completo su [Bloccante vs Non Bloccante][]. 
+
+---
+
+Node è simile in design, e influenzato, da sistemi come la [Event Machine][] di Ruby o [Twisted][] di Python. Node porta il modello ad eventi un po' oltre. Presenta un [loop di eventi][] come un costrutto a runtime invece che come libreria. In altri sistemi c'è sempre una chiamata bloccante per chiamare un Event Loop.
+
+Normalmente il comportamento viene definito tramite callbacks all'inizio di uno script e alla fine viene avviato un server tramite una chiamata bloccante come `EventMachine::run()`.
+In Node non c'è un loop d'avvio dell'evento come quello. 
+Node semplicemente entra nell'event loop dopo aver eseguito lo script di input. 
+Node esce dal loop d'evento quando non ci sono più callbacks da eseguire.
+Questo comportamento è simile al JavaScript dei browser - il loop d'eventi è nascosto all'utente.
+
+HTTP è la prima classe abitante in Node, pensata con l'idea di streaming e bassa la tenza in mente.
+Questo rende Node adatto per la creazione di una libreria web o di un framework.
```

Comment:
I think we could translate this sentence with `Questo rende Node adatto al la creazione di una libreria web o di un framework.`